### PR TITLE
Charts: provide an accessible title and description

### PR DESCRIPTION
**Pull request by dlabrecq**
Victory provides the ability to add a description and title to SVGs for accessibility. However, users must override `containerComponent` just to set these properties -- it's not clear how to provide accessibility.

In order to make this more intuitive, I've added `ariaDesc` and `ariaTitle` properties to our top level components; `Chart`, `ChartGroup`, etc. The property names we chosen because `ChartDonut` and `ChartDonutUtilization` already have `title` and `subTitle` properties; thus, avoiding a breaking change.

Note that these top-level properties just pass through to the container's `desc` and `title` props. The underlying SVG refers to `desc` and `title` tags via the SVG's aria-labeledby attribute. 

The approach allows users to continue using the `allowZoom` prop of `Chart` and `ChartGroup` without overriding `containerComponent`. If the user must override `containerComponent`, the `allowZoom` property would not be able to provide a default `VictoryZoomContainer` component. 

These top-level properties also help to set `desc` and `title` for `ChartPie`, `ChartDonut` etc. Overriding `containerComponent` would not work in this particular case because the chart is wrapped with a `ChartContainer` component and rendered `standalone=false` -- required to show title, subTitle, and legend within the same SVG. The `desc` and `title` properties must be applied to the outermost container.

Example output:
Screen Shot 2019-07-11 at 2 01 34 AM

Fixes https://github.com/patternfly/patternfly-react/issues/1498

### CODE REVIEW

**Comment by redallen** [comment]
Conflicts with #2497 which just went in.

**Comment by tlabaj** [comment]
@dlabrecq can you please link associated issue. Thanks

**Comment by dlabrecq** [comment]
Thanks @jschuler . I've renamed those properties as `ariaDesc` and `ariaTitle`.

File: packages/patternfly-4/react-charts/src/components/Chart/Chart.tsx [hunk 1]
```diff
@@ -349,6 +365,7 @@ export const Chart: React.FunctionComponent<ChartProps> = ({
 
   // destructure last
   theme = getTheme(themeColor, themeVariant),
+  containerComponent = allowZoom ? <VictoryZoomContainer /> : <ChartContainer theme={theme} />,
```
Comment (redallen) [inline packages/patternfly-4/react-charts/src/components/Chart/Chart.tsx]: Can we not pass `desc: ariaDesc, title: ariaTitle,` here and avoid `React.cloneElement(containerComponent as React.ReactElement<any>` later?

Comment (dlabrecq) [reply packages/patternfly-4/react-charts/src/components/Chart/Chart.tsx]: Don't want to modify a given container component directly. I could add the props here, but would be adding it again later in the code

Comment (dlabrecq) [reply packages/patternfly-4/react-charts/src/components/Chart/Chart.tsx]: Didn't want to modify a given containerComponent directly; thus, the clone. Could add props to the defaults while destructuring, but still need to add the same props while cloning the given component. Keeping it simple and following Victory, which also clones the containerComponent, labelComponent, etc.

**Comment by redallen** [comment]
@mattnolting Created relevant issue and followup PR for you.

File: packages/patternfly-4/react-charts/src/components/ChartArea/examples/ChartArea.md [hunk 2]
```diff
@@ -163,6 +167,8 @@ class MultiColorChart extends React.Component {
       <div ref={this.containerRef}>
         <div className="area-chart-legend-bottom-responsive">
           <Chart
+            ariaDesc="Average number of pets"
+            ariaTitle="Donut chart example"
```
[the remark this answers is not in the document]
Comment (dlabrecq) [reply packages/patternfly-4/react-charts/src/components/ChartArea/examples/ChartArea.md]: Thank you. I'll create a PR to fix the title text.

**Pull request merged**